arch/xtensa/src/esp32/esp32_wireless.c

```diff
@@ -86,6 +89,85 @@ static uint32_t *g_phy_digital_regs_mem = NULL;
 
 static bool g_is_phy_calibrated = false;
 
+#ifdef CONFIG_ESP32_PHY_INIT_DATA_IN_PARTITION
+static const char phy_partion_label[] = "phy_init";
+#endif
+
+#ifdef CONFIG_ESP32_SUPPORT_MULTIPLE_PHY_INIT_DATA
+
+static phy_init_data_type_t g_phy_init_data_type = 0;
+
+static phy_init_data_type_t g_current_apply_phy_init_data = 0;
+
+static char g_phy_current_country[PHY_COUNTRY_CODE_LEN] =
+  {
+    0
+  };
```

Comment:
Optional
```suggestion
static char g_phy_current_country[PHY_COUNTRY_CODE_LEN];
```
or remove 2 spaces indentation.